woocommerce/woocommerce-gateway-stripe
# Remove the "Where are the customization options?" notice

## Changes proposed in this Pull Request:

Remove the "Where are the customization options?" notice from the Settings page.

It reads...

> Where are the customization options? In the new checkout experience, payment method details are automatically displayed in your customers’ languages so you don’t have to worry about writing them manually.

We're removing it since this is no longer relevant. We introduced the customization options for the updated checkout experience.

| Settings tab | Payment methods tab |
|--------|--------|
| <img width="700" alt="image" src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/41606954/bb3b41d1-1be9-4cc0-9aa7-5529c894e7f8"> |<img width="700" alt="image" src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/41606954/38f8faf1-9cf2-40cb-9e71-8a49993c6433"> |

## Testing instructions

- Check out `develop`
- Go to the Stripe settings page > both the Settings or Payment methods tab. At `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=methods`
- Confirm you can see the "Where are the customization options?" notice
    If you dismissed it before, it may be easier to set up a new website. It was not showing on my local site even after removing the dismissed option from the db for some reason
- Check out this branch - or install build of this branch in your test site
- Go to the Stripe settings page > both the Settings or Payment methods tab
- Confirm that:
    - The notice isn't displayed in either tab
    - There are no errors in the browser console

---


**Post merge**
